force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -264,6 +264,13 @@
         <shortDescription>No Service Participants Created Warning</shortDescription>
         <value>No Contacts are selected, so no Service Participants will be created.</value>
     </labels>
+    <labels>
+        <fullName>No_Service_Sessions_Created_Warning</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>No Service Sessions Created Warning</shortDescription>
+        <value>No Sessions were added, so no Service Sessions will be created.</value>
```

Comment:
@lbeller can you please review this new label, similar to the label we already had above on line 265. Thank you!

Reply:
Can we just lower case sessions in the first part of the sentence (No sessions were added, so no Service Sessions will be created.)

Reply:
@lbeller - change has been made. Thank you!